example/android/app/src/main/java/com/example/MainApplication.java

```diff
@@ -25,8 +24,7 @@ protected boolean getUseDeveloperSupport() {
     @Override
     protected List<ReactPackage> getPackages() {
       return Arrays.<ReactPackage>asList(
-          new MainReactPackage(),
-          new RNSvgPackage()
```

Comment:
Well, on the library source itself we have some rudimentary jest snapshot tests that are set up that can be run with `npm test` (see `src/__tests__, src/__mocks__, ./circle.yml, ./package.json`) but I think what you are asking above is automated tests on the example app. Unfortunately, we don't have anything setup right now for the example but would love to add something. Ironically, the src tests are really just duplicates of portions of the example app. Jest snapshot tests couple probably work specifically for the example app. There is a ton of room for improvement in both the src tests and getting something started on the example app. I would welcome any contributions that you or others are willing to contribute in a separate PR(s).